server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/AutoDbMigration.java

```diff
@@ -75,6 +77,27 @@ void installH2() {
   @VisibleForTesting
   protected void createSchema(Connection connection, String dialectId) {
     DdlUtils.createSchema(connection, dialectId, false);
+    hackFixForProjectMeasureTreeQueries(connection);
+  }
+
+  private void hackFixForProjectMeasureTreeQueries(Connection connection) {
```

Comment:
This hack can't be understand without the context. A javadoc or a link to the JIRA ticket could help.

Btw I'm quite disappointed that you didn't write a test !